bpf/http_sock.c

```diff
@@ -13,15 +13,15 @@ char __license[] SEC("license") = "Dual MIT/GPL";
 
 // Temporary tracking of accept arguments
 struct {
-    __uint(type, BPF_MAP_TYPE_HASH);
+    __uint(type, BPF_MAP_TYPE_LRU_HASH);
```

Comment:
I changed some of these because kretprobes may not fire, so we can't be sure we'll clean-up these maps.